releases/v1.1-alpha.20170831.md

```diff
@@ -0,0 +1,104 @@
+---
+title: What&#39;s New in v1.1-alpha.20170831
+toc: false
+summary: Additions and changes in CockroachDB version v1.1-alpha.20170831
+---
+
+## Aug 31, 2017
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170831.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170831.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170831.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170831.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### SQL Language Changes
+
+- New command `IMPORT CSV` is now available and provides the fastest way to import large amounts of data into CockroachDB. This command is only available in builds that include CCL-licensed code, but using it does not require an enterprise license. TODO(jseldess): work with product/marketing on this language [#17539](https://github.com/cockroachdb/cockroach/pull/17539)
+- The session variable `sql_safe_updates` can be set to disable certain "dangerous" SQL statements, such as `DELETE` or `UPDATE` without a `WHERE` clause. It defaults to true for the `cockroach sql` shell. [#17604](https://github.com/cockroachdb/cockroach/pull/17604)
```

Comment:
It won't default to true until you greenlight cockroachdb/cockroach#17889.

Reply:
Removed that second sentence, since it's not possible for that open PR to make it in.